lambda-functions/notification-template-formatter-service/src/main/resources/permissionAlertEmailTemplate.html

```diff
@@ -43,22 +43,11 @@
 
 <table style="width:800px">
 
-    <tr>
-        <td class="rowkey">Asset Type</td>
-        <td>${assetType}</td>
-    </tr>
     <tr>
         <td class="rowkey">Source</td>
-        <td>${cloudType}</td>
-    </tr>
-    <tr>
-        <td class="rowkey">Error Message</td>
-        <td>${permission}</td>
-    </tr>
-    <tr>
-        <td class="rowkey">Account</td>
-        <td>${accountNumber}</td>
+        <td>${source}</td>
     </tr>
+    ${permissionIssues}
 </table>
```

Comment:
Please reformat html file.

Reply:
Addressed in commit

Reply:
Table and body should not be in the same indentation level.